Makefile

```diff
@@ -117,7 +118,7 @@ test: vendor
 
 .PHONY: integration
 integration: $(MINISHIFT_BINARY)
-	go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --binary $(MINISHIFT_BINARY)
+	go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --binary $(MINISHIFT_BINARY) --tags "$(TEST_TAGS)"
```

Comment:
does that not force me to always specify a tag? Should there not be a way run all tests w/o specifying a tag? Trying to understand your thinking here.